cmd/print.go

```diff
@@ -235,6 +286,9 @@ func newPrintCommand() *cobra.Command {
 	cmd.Flags().StringVarP(&pr.outputFormat, "output", "o", "yaml",
 		fmt.Sprintf(`Output format. One of: (%s)`, strings.Join(allowedFormats, ", ")))
 
+	cmd.Flags().StringVarP(&pr.outputFile, "output-file", "", "",
+		"Path to the  conversion result file, the tool will output the conversion result to the file. Value pattern should be *.yaml of *.json. If not set, the tool will output to stdout")
```

Comment:
we should not enforce any specific validation on the file extension. Users should be able to use whatever file they prefer